src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/UpdateAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -31,6 +32,23 @@ public class SetAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
             ValueFromPipeline = true,
             HelpMessage = "The VirtualNetworkGatewayConnection")]
         public PSVirtualNetworkGatewayConnection VirtualNetworkGatewayConnection { get; set; }
+        
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Whether to establish a BGP session over a S2S VPN tunnel")]
+        public string EnableBgp { get; set; }
```

Comment:
@henry416 if this cmdlet is a PATCH operation, this parameter should be a bool, if it's a PUT operation, then it should be a SwitchParameter. Same comment about not parsing a string to a bool.